WooCommerce/Classes/ViewRelated/Orders/Cells/OrderNoteTableViewCell.swift

```diff
@@ -80,25 +76,35 @@ private extension OrderNoteTableViewCell {
     /// Updates the Status Label + Icon's Color
     ///
     func noteTypeWasUpdated() {
+        let theAuthor = author ?? ""
+        let dateOfCreation = dateCreated ?? ""
         if isCustomerNote {
             iconButton.backgroundColor = StyleManager.statusPrimaryBoldColor
-            statusLabel.text = NSLocalizedString("Note to customer", comment: "Labels an order note to let user know it's visible to the customer")
+            let template =
+                NSLocalizedString("%@ - %@ (To Customer)",
```

Comment:
nit: just wondering if this is the only template without the order in this class?

Reply:
I noticed that as well. Fixed!